### ISSUE TITLE
Weird handling of Fastq data

### ISSUE THREAD

**Issue by standage** [ged-lab/khmer#817]
I grabbed these two interleaved read pairs from one of my data sets to do some screed testing and see if I can begin to wrap my head around #606. I can't remember if these are from a HiSeq or a GAII, but I think it's the former.

```
@HWI-ST412:261:d15khacxx:8:1101:3149:2157 1:N:0:ATCACG
GTACACTATAGTGAGATTCCTTATACCTCGCAATATGATCTCCTGTCGTTACGTCCATTGCTCTTAACAAAGAACACTCATAGAGTAGAATCGAAAAAAC
+
@@CDAFFAHFACC?DBHIAIIICGC?AHFEEHGGIIGG>CEDHBDHDGBGGBB8B<FGHG4@GEFDA@.7CC3?)7@?CDECCAA663;;;(;=??B?>5
@HWI-ST412:261:d15khacxx:8:1101:3149:2157 2:N:0:ATCACG
TCGTTTTCTTGTAGATTAGATTTGACATAAATGTCAATTTTTTTGTATCTACCGTCACTTGAATACTCAGTTGCAATATAAT
+
;+=D7D?DHGHFDCAE@BCCACHHGFBHICC<EHE?BFBDHGGIIADDB*?D8@B@FG>FCE;@D:A>EECEFF3?BD@(.6
@HWI-ST412:261:d15khacxx:8:1101:3426:2112 1:N:0:ATCACG
CTAAACACAGCAAATTTGTCAGAGATTTAATCCGTGAAGTAACTGGTCATGCTCCTTATGAAAAACGTGCTATGGAATTGCTCAAAGTATCACAGGATA
+
1=DDFFAFFFHFIGBHGGDHIH>GC9?FEHGFGIIIIEEBGGBFHGGEIGHICHBFGCHEGIIIIGGCEGEHCHA?@@B?@D@C@>@>CCA>(;5;AA>
@HWI-ST412:261:d15khacxx:8:1101:3426:2112 2:N:0:ATCACG
GCCCAGCCTTCTTTTCAAAAACTTCAATGCACGCTTATCCTTTGATACTTTGAGCAATTCCATAGCACGTTTTTCATAAGGAGCATGACCAGTTACTTCA
+
@?@A=BDD;DF3CFDHGEEHI:AFAHGII>FBCFE):B<B<DDHA<FGIHHFG=GGIGCGEIGGF@A=B2=7?1=;3?;7.;;=AC@CCCDDA>;:;@C@
```

I can't even get scripts like ``split-paired-reads.py`` and ``interleave-reads.py`` to handle data in this format correctly. The split script won't split them because they aren't `.*/[12]` terminated. So I manually split them, and then the interleave script writes them out in Fasta format, lops of the end of the first line, and appends a `/1` or `/2` to designate with mate it is.

Apologies if these are known issues, but I can't imagine this is the intended behavior. Please let me know if I'm missing something.

**Comment by mr-c** [comment]
This is the  Casava 1.8+ format; alas we've known about this problem for a long time: #23 

But it is ripe for fixing if you wanted to!

**Comment by ctb** [comment]
Actually, there's some other serious breakage here... working on it.

**Comment by ctb** [comment]
Fixed in #818.
